tck/api/src/main/java/org/eclipse/microprofile/metrics/tck/MetricRegistryTest.java

```diff
@@ -56,24 +51,6 @@ public class MetricRegistryTest {
     @Metric(name = "nameTest", absolute = true)
     private Counter nameTest;
 
-    @Inject
-    private Counter countTemp;
-
-    @Inject
-    private Histogram histoTemp;
-
-    @Inject
-    private Timer timerTemp;
```

Comment:
not clear why we're removing histogram and timer test

Reply:
This test was testing against the _registration_ method that allowed for users to provide an implementation of a Metric. It was using the  injected metric instances here  and then registering it under a new name below.